aws/sdk-codegen/build.gradle.kts

```diff
@@ -36,7 +36,7 @@ val generateAwsRuntimeCrateVersion by tasks.registering {
     val resourcesDir = "$buildDir/resources/main/software/amazon/smithy/rustsdk"
     val versionFile = file("$resourcesDir/sdk-crate-version.txt")
```

Comment:
Is this file even used?

Reply:
Good eyes. No it isn't. In fact, the entire `generateAwsRuntimeCrateVersion` isn't used (ran our release pipeline and confirmed it went ok without it). Removed it in 73b1e81.